### PR TITLE
Fixes for grammar and clarity

### PR DESCRIPTION
**Pull request by brson**
r? @Hoverbear cc @siddontang

### CODE REVIEW

**Comment by siddontang** [comment]
Cool, PTAL @tikv/pingcap-i18n

File: src/key-value-engine/B-Tree-vs-Log-Structured-Merge-Tree.md [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
+In general, there are three critical metrics to measure the performance of a data structure: write amplification, read amplification, and space amplification. This section aims to describe these metrics. 
 
-For hard disk drives (HDDs), the cost of disk seek is enormous, so that the performance of random read/write is worse than that of sequential read/write. This article assumes that flash-based storage is used so we can ignore the cost of disk seeks.
+For hard disk drives (HDDs), the cost of disk seek is enormous, such that the performance of random read/write is worse than that of sequential read/write. This article assumes that flash-based storage is used so we can ignore the cost of disk seeks.
 
 ### Write Amplification
 
-`Write amplification` is the ratio of the amount of data written to the storage device versus the amount of data written to the database. 
+_Write amplification_ the ratio of the amount of data written to the storage device versus the amount of data written to the database.
```
Comment (CaitinChen) [inline src/key-value-engine/B-Tree-vs-Log-Structured-Merge-Tree.md]:
```suggestion
_Write amplification_  is the ratio of the amount of data written to the storage device versus the amount of data written to the database. 
```

**Review by CaitinChen — COMMENTED** [review]
Rest LGTM

File: src/key-value-engine/B-Tree-vs-Log-Structured-Merge-Tree.md (hunk 1 above)
Comment (brson) [reply src/key-value-engine/B-Tree-vs-Log-Structured-Merge-Tree.md]: Good catch. I've fixed it.

**Comment by brson** [comment]
Thanks for the review @CaitinChen. I've pushed a fix.

**Comment by CaitinChen** [comment]
@brson No problem ^_^

**Review by Hoverbear — APPROVED** [review]
LGTM, thanks Brian!

**Pull request merged**